src/script.js

```diff
@@ -0,0 +1,26 @@
+function validatePassword() {
+    let inputFieldEl = document.getElementById("inputField").value
+
+    let passwordArray = inputFieldEl.split("")
```

Comment:
Overall since this is so concise already, I'm thinking you don't even need to split this into an array. You can test the length on a string as well, so since you are using inputFieldEl for your other tests, you can also test inputFiedlEl.length() and should get the same result.